src/bindings/event.ts

```diff
@@ -35,11 +37,15 @@ export const bindEvents = async (context: Context): Promise<void> => {
   if (!logger) {
     return;
   }
+
   logger.info(`Config loaded! config: ${JSON.stringify({ price: botConfig.price, unassign: botConfig.unassign, mode: botConfig.mode, log: botConfig.log })}`);
   const allowedEvents = Object.values(GithubEvent) as string[];
-  const eventName = `${name}.${payload.action}`;
+  const eventName = payload.action ? `${name}.${payload.action}` : name; // some events wont have actions as this grows
```

Comment:
looks like a nice catch!

Reply:
@0xcodercrane should we make an issue for this? I do not have enough context on this issue to create it myself if needed.